tools/build/Makefile.rules

```diff
@@ -405,3 +405,11 @@ $(KCONFIG_AUTOHEADER): $(DEPENDENCY_CACHE) $(KCONFIG_CONFIG) $(obj)/conf
 	$(Q)echo "     "GEN"   "$@
 	$(Q)mkdir -p $(KCONFIG_INCLUDE)config $(KCONFIG_INCLUDE)generated
 	$(Q)$(obj)/conf -s --silentoldconfig Kconfig
+
+include $(top_srcdir)bindings/python/bindings.mk
```

Comment:
I'm trying to centralize the place where I will have to modify variables in the future.  If I add a cython source file, absent this central include, I'm going to have to edit Makefile.rules and bindings/python/Makefile with mostly the same variables.